ocean/nemo/nemolite2d/kernels/fortran/continuity_mod.f90

```diff
@@ -180,10 +178,10 @@ end subroutine invoke_continuity_arrays_basic
 
   subroutine continuity_code(ji, jj,                     &
                              ssha, sshn, sshn_u, sshn_v, &
-                             hu, hv, un, vn, rdt, e12t)
```

Comment:
Please could you ensure that the various places that call this kernel (in e.g. the manual versions) are updated to remove the rdt argument. e.g. psykal_acc/time_step_mod.f90

Reply:
True, I updated some. But note that the one you pointed on `psykal_acc/time_step_mod.f90` has its own redefined `continuity_code`.

Reply:
Thanks.